spec/features/messages_spec.rb

```diff
@@ -70,6 +70,10 @@
       Settings.assignments = assignments_settings
     end
 
+    after do
+      Settings.assignments = nil
```

Comment:
Wäre es nicht safer, die Einstellungen von vor dem Test wiederherzustellen? Im Moment mag das im Core `nil` sein, aber vielleicht ändert sich das ja mal in Zukunft, und dann wäre wieder ein flaky Spec da.
Achtung: Beim Wegspeichern unbedingt `.clone` verwenden, sonst bringts unter Umständen nichts, wie wir hier schon bemerkt haben: https://github.com/hitobito/hitobito/blob/master/spec/models/event_spec.rb#L860